platform/vs/docker-sonic-vs/orchagent.sh

```diff
@@ -6,8 +6,18 @@ else
     export platform=$fake_platform
 fi
 
-CFG_VARS=$(sonic-cfggen -d --var-json 'DEVICE_METADATA')
-MAC_ADDRESS=$(echo $CFG_VARS | jq -r '.localhost.mac')
+EXIT_SWSS_VARS_FILE_NOT_FOUND=1
+SWSS_VARS_FILE=/etc/sonic/swss_syncd_vars.j2
+
+if [ ! -f "$SWSS_VARS_FILE" ]; then
+    echo "SWSS vars template file not found"
+    exit $EXIT_SWSS_VARS_FILE_NOT_FOUND
+fi
+
+# Retrieve SWSS vars from sonic-cfggen
+SWSS_VARS=$(sonic-cfggen -d -y /etc/sonic/sonic_version.yml -t $SWSS_VARS_FILE)
```

Comment:
>$SWSS_VARS_FILE [](start = 63, length = 15)

If SWSS vars template file not found, the command substitution will fail, so you can catch the error and exit. It is too verbose to check every input files.

Reply:
Removed the verbose check for the template file.